README.md

```diff
@@ -187,7 +185,14 @@ See the [getting started](https://meshery.io/#getting-started) section to quickl
 <hr />
 <p style="clear:both;">&nbsp;</p>
 
-<a href="https://docs.google.com/presentation/d/14kxjwYSJ_FyE3K_6CDEd6oq2kqwn0OSE8RDJ4H-KlKU/edit?usp=sharing"><img alt="Meshery Logo" src=".github/assets/images/meshery/meshery-logo-dark-text-side.png"  style="margin-left:10px; margin-bottom:10px;" width="45%" align="left"/></a>
+<a href="https://meshery.io">
+<picture><img src="./.github/assets/images/readme/meshery-logo-light-text-side.svg#gh-dark-mode-only"
+style="margin:10px;" width="45%" 
+alt="Meshery - the Cloud Native Manager" align="left" />
+<img src="./.github/assets/images/readme/meshery-logo-dark-text-side.svg#gh-light-mode-only"
+style="margin:10px;" width="%" 
+alt="Meshery - the Cloud Native Manager" align="left" /></picture></a>
```

Comment:
Please use ```<picture>``` tag something like

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/meshery/meshery/blob/master/.github/assets/images/Meshery-celebration-Dark.svg">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/meshery/meshery/blob/master/.github/assets/images/Meshery-celebration-Light.svg">
  <img alt="Meshery Logo" src="">
  </picture>
```

The ```#gh-dark-mode-only``` thing doesn't work outside of the GitHub, since GitHub readme's are used in NPM package page, for example.

Reply:
It is officially deprecated by GitHub https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

Reply:
@anshgoyalevil please have a look. I have updated the format of picture element.

Reply:
@anshgoyalevil  and @vishalvivekm  how do we add raw.githubusercontent... to my issue as I dont have these images already on the master branch? I am just adding them in .github/assets/images/readme on this branch.

Reply:
@anshgoyalevil I have added raw.githubcontent but logo is not being displayed.Apparently, it is because it is not in master branch yet.